plugins/default/lib/test_mongodb_health.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+import unittest
+
+from mongodb_health import analize, parse_mongo_result, report
+
+
+class TestAnalize(unittest.TestCase):
+
+    def runTest(self):
+        self.assertEqual(analize(1), 0)
+        self.assertEqual(analize(2), 0)
+        self.assertEqual(analize(0), 1)
+        self.assertEqual(analize(7), 1)
+        self.assertEqual(analize(3), 2)
+        self.assertEqual(analize(5), 2)
+        self.assertEqual(analize(7), 1)
+        self.assertEqual(analize(9), 2)
+        self.assertEqual(analize(None), 3)
+
+
+class TestParseMongoResult(unittest.TestCase):
+
+    def runTest(self):
+        self.assertEqual(parse_mongo_result("0\n"), 0)
+        self.assertEqual(parse_mongo_result("1\n"), 1)
+        self.assertEqual(parse_mongo_result("2\n"), 2)
+        self.assertEqual(parse_mongo_result("3\n"), 3)
+        self.assertEqual(parse_mongo_result("5\n"), 5)
+        self.assertEqual(parse_mongo_result("7\n"), 7)
+        self.assertEqual(parse_mongo_result("9\n"), 9)
+        self.assertEqual(parse_mongo_result("10\n"), 10)
+        self.assertEqual(parse_mongo_result("\n"), None)
+        self.assertEqual(parse_mongo_result(""), None)
+
+
+class TestReport(unittest.TestCase):
+
+    def runTest(self):
+        self.assertEqual(report(
+            ("mongodb-1-1-s1ngl",),
+            (1,),
+            (0,)), 0)
```

Comment:
I like the naming parameters, so will do this as well.  As far as the result from a single node mongo, unfortunately when it comes to automatically created openshift mbaases, it only includes a single node and so does our core template currently.  I do feel like we could justify warning on this, but I found myself thinking that if we do that we should also warn when a mongo pod doesn't have persistent storage, but that opens a whole new can of worms beyond the point of this check.  Checked with the rest of team and seems to be agreement that single node should return nagios.OK at least for now.